package.json

```
@@ -48,58 +48,45 @@
     }
   },
   "devDependencies": {
-    "@silverstripe/eslint-config": "0.0.5",
-    "@silverstripe/webpack-config": "^1.5.0",
+    "@babel/runtime": "^7.20.0",
+    "@silverstripe/eslint-config": "1.0.0-alpha6",
+    "@silverstripe/webpack-config": "^2.0.0-alpha9",
     "@storybook/addon-actions": "^3.4.11",
     "@storybook/addons": "^3.4.11",
     "@storybook/react": "^3.4.11",
-    "babel-jest": "^23.6.0",
+    "babel-jest": "^29.2.2",
     "copy-webpack-plugin": "^4",
-    "jest-cli": "^23.6.0",
-    "webpack": "^2.6.1"
+    "jest-cli": "^29.2.2",
+    "jest-environment-jsdom": "^29.3.1",
+    "webpack": "^5.74.0",
+    "webpack-cli": "^5.0.0"
   },
   "dependencies": {
-    "apollo-client": "^2.4.2",
-    "apollo-link-batch-http": "^1.2.1",
-    "babel-core": "^6.26.3",
-    "babel-polyfill": "6.7.4",
-    "babel-runtime": "6.26.0",
-    "bootstrap": "^4.3.1",
+    "@apollo/client": "^3.7.1",
+    "bootstrap": "^4.6.2",
+    "core-js": "^3.26.0",
     "classnames": "^2.2.5",
-    "deep-freeze-strict": "^1.1.1",
-    "dropzone": "^5.5.1",
-    "graphql": "^0.13.2",
-    "graphql-fragments": "^0.1.0",
-    "graphql-tag": "^2.8.0",
-    "griddle-react": "^0.8.2",
-    "jquery": "^3.1.1",
-    "merge": "^1.2.1",
-    "modernizr": "^3.5.0",
-    "popper.js": "^1.14.4",
-    "prop-types": "^15.6.2",
-    "qs": "^6.2.1",
-    "react": "^16.6.1",
-    "react-apollo": "^2.1.0",
+    "prop-types": "^15.8.1",
+    "qs": "^6.11.0",
+    "react": "^18.2.0",
     "react-dnd": "^5.0.0",
     "react-dnd-html5-backend": "^5.0.1",
-    "react-dom": "^16.6.1",
-    "react-redux": "^5.0.7",
-    "react-router-dom": "^4.4.0-beta.6",
-    "react-selectable": "^2.0.2",
-    "reactstrap": "^6.4.0",
-    "redux": "^4.0.0",
-    "redux-form": "^7.4.2",
-    "redux-logger": "^2.4.0",
-    "redux-thunk": "^1.0.3",
-    "webpack-sources": "^1.1.0"
+    "react-dom": "^18.2.0",
+    "react-load-script": "^0.0.6",
+    "react-redux": "^8.0.4",
+    "react-router": "^6.4.2",
+    "react-router-dom": "^6.4.2",
+    "react-select": "^5.5.8",
+    "reactstrap": "^8.9.0",
+    "redux": "^4.2.0",
+    "redux-form": "^8.3.8",
+    "redux-thunk": "^2.4.1",
+    "webpack-sources": "^3.2.3"
   },
```

```suggestion
  },
  "browserslist": [
    "defaults"
  ],
```
Make sure we're explicitly stating the browserset we support here